nx.json

```diff
@@ -134,6 +134,11 @@
         "fdp"
       ],
       "implicitDependencies": ["core"]
+    }, 
+    "moment-adapter": {
+      "tags": [
+        "fdm"
+      ]
```

Comment:
implicitDependencies: ["core-datetime"]?

Reply:
also can not build moment-adapter, getting the next error:
`Cannot convert undefined or null to object`

Reply:
I have problem with this and need some help.
![image](https://user-images.githubusercontent.com/5276786/126729011-f2f32727-3893-4b5e-b9aa-09cf9019d9f9.png)

Actually this is pretty strange, because method are not abstract and DateLocale are already at export